hooks/pre-command

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+aws_account_id="697149045717"
```

Comment:
Do you think this ID can be customised?

Reply:
https://github.com/elastic/oblt-aws-auth-buildkite-plugin/pull/1/commits/30574d2f09e90491e8688c9fe6ca983189938954